ext/vscode/package.json

```diff
@@ -364,7 +370,81 @@
                     }
                 ]
             }
-        }
+        },
+        "walkthroughs": [
+            {
+                "id": "azd.start",
```

Comment:
Is this id what will come through in telemetry? Or unrelated?

Does this have telemetry attached to it?

Reply:
I believe there is core VSCode telemetry for walkthroughs. The short ID is to get around length limits on treatment variables in EXP.

Reply:
Yep, I confirmed with Harald, VSCode will capture it for us.

Reply:
Great!